backend/database/migrations/2020_10_02_073030_create_role_user_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRoleUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('role_user', function (Blueprint $table) {
+            $table->foreignId('role_id')->constrained();
```

Comment:
Maybe you can add a default value for role_id when creating the RoleUser. Because we agreed if the user is newly created it must be a Customer and then the Admin can change it.